### DISCUSSION TITLE
Please add looppointer

### DISCUSSION THREAD

**Discussion by guettli** [golangci/golangci-lint#4052] [Q&A] [answered]
Please add looppointer.

I have seen the open issue, and don't understand why this got rejected up to now.

https://github.com/golangci/golangci-lint/issues/1404

The looppointer issue is very common in Go.

**Comment by ldez** [comment, marked as answer]
The discussion about looppointer has happened here:
- https://github.com/golangci/golangci-lint/issues/1041
- https://github.com/golangci/golangci-lint/pull/1924
- https://github.com/golangci/golangci-lint/issues/1404
